plutus-core/plutus-ir/src/PlutusIR/Transform/Inline.hs

```diff
@@ -53,18 +52,15 @@ We differ from the paper a few ways, mostly leaving things out:
 Functionality
 ------------
 
-PreInlineUncoditional: we don't do it, since we don't bother using usage information.
+PreInlineUnconditional: we don't do it, since we don't bother using usage information.
 We *could* do it, but it doesn't seem worth it. We also don't need to worry about
 inlining nested let-bindings, since we don't inline any.
 
-CallSiteInline: not worth it.
+CallSiteInline: we do it based on ambient arity, not context. (TODO in PLT-1146)
 
 Inlining recursive bindings: not worth it, complicated.
 
-Context-based inlining: we don't do CallSiteInline, so no point.
-
-Beta reduction: not worth it, but would be easy. We could do the inlining of the argument
```

Comment:
I think this is outdated? We do it in [Beta.hs](https://github.com/input-output-hk/plutus/blob/5c2ccce330f5d1680956945bccb435ae6e02ece3/plutus-core/plutus-ir/src/PlutusIR/Transform/Beta.hs#L1)?

Reply:
Yes, but say that - the point is that we don't do it here.

Reply:
OK I'll mention we do that in Beta.hs, not here then.